src/lib/holocene/button.svelte

```diff
@@ -100,24 +100,30 @@
   export let trailingIcon: IconName = null;
   export let count = 0;
   export let id: string = null;
+  export let icon: IconName = null;
   export let href: string = null;
   export let target: string = null;
   export let active = false;
 </script>
 
 {#if href && !disabled}
-  <Link
-    {target}
+  <a
```

Comment:
I think we'll want to copy the behavior of Link on the on:click instead of using native <a> behavior. An <a> does a hard refresh navigation (triggers all load functions) instead of doing client-side navigation where it pushes to history.

  const onLinkClick = (e: MouseEvent) => {
    // Skip if middle mouse click or new tab
    if (e.button === 1 || target || e.metaKey) return;
    e.preventDefault();
    e.stopPropagation();
    goto(href);
  };
</script>

<a
  {href}
  target={newTab ? '_blank' : null}
  rel={newTab ? 'noreferrer' : null}
  class={merge('link', icon ? 'inline-flex' : 'inline', className)}
  class:active
  on:click={onLinkClick}
  tabindex={href ? null : 0}
  {...$$restProps}
>

Reply:
Okay, cool! I added it in and then removed it when I was switching it over. I honestly wasn't sure what the difference was. Thanks for the explanation Alex! I'll add this in now :)